vxlanBridge.go

```diff
@@ -221,6 +222,18 @@ func (self *Vxlan) AddLocalEndpoint(endpoint OfnetEndpoint) error {
 	// Save the flow in DB
 	self.macFlowDb[endpoint.MacAddrStr] = macFlow
 
+	// NOTE: Remove this check when EVPN support is added
+	if !VXLAN_GARP_SUPPORTED {
```

Comment:
It'll be better to guard just sendGARP() with this check in case we need to add more code below